apps/core/wagtail_hooks.py

```diff
@@ -0,0 +1,19 @@
+from wagtail.contrib.modeladmin.options import ModelAdmin, modeladmin_register
+
+from apps.core.models.feedback import Feedback
+
+
+class FeedbackAdmin(ModelAdmin):
+    model = Feedback
+    base_url_path = "feedbackadmin"
+    menu_label = "Feedback"
+    menu_icon = "pilcrow"
+    add_to_settings_menu = False
+    exclude_from_explorer = False
+    add_to_admin_menu = True
```

Comment:
We can drop these:
```
add_to_settings_menu = False
    exclude_from_explorer = False
    add_to_admin_menu = True
    ```
    as those are the default values.
    
    We can also drop `menu_label = "Feedback"` and add to the model:
    
    ```
    class Meta:
         verbose_name="Feedback"
         verbose_name_plural="Feedback items"
    ```
    
This wil give nicer labels in the admin.